src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEval.java

```diff
@@ -126,125 +123,27 @@
 )
 @DocumentedFeature
 @BetaFeature
-public class VariantEval extends MultiVariantWalker {
+public class VariantEval extends MultiVariantWalkerGroupedOnStart {
     public static final String IS_SINGLETON_KEY = "ISSINGLETON";
 
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             doc="File to which variants should be written")
     protected File outFile;
 
-    /**
-     * The variant file(s) to evaluate.
-     */
-    @Argument(fullName="eval", shortName = "eval", doc="Input evaluation file(s)", optional=false)
-    public List<FeatureInput<VariantContext>> evals;
```

Comment:
Yes - this was trying to make a compromise and move toward the goal of creating the standalone VariantEvalEngine.

What about letting the ArgumentCollection encapsulate some of logic around creating DrivingVariants? In theory one might want different walkers based around this, and it would be helpful to have one piece of code that does that? Or do you think the better pattern is to make the VariantEvalEngine class that takes the ArgumentCollection in the constructor, and then perhaps that provides methods to set up DrivingVariants?